tests/test_imageIo1.py

```diff
@@ -191,6 +191,48 @@ def testBBoxFromMetadata(self):
                                                        lsst.geom.Extent2I(145, 200)),
                                        hdu=hdu)
 
+    @unittest.skipIf(dataDir is None, "afwdata not setup")
+    def testReadFitsWithOptions(self):
+        xy0Offset = lsst.geom.Extent2I(7, 5)
+        bbox = lsst.geom.Box2I(lsst.geom.Point2I(10, 11), lsst.geom.Extent2I(31, 22))
+
+        with tempfile.NamedTemporaryFile() as f:
```

Comment:
Why not `TestCase.getTempFilePath`?

Reply:
OK. I have a mild preference for standard library functions when they do the job, but I admit it's a bit skanky to open the file twice (though it is documented to work on unix).